docker-compose.yml

```diff
@@ -7,17 +7,19 @@ services:
     command: lighttpd -D -f /etc/lighttpd/lighttpd.conf
     volumes:
       - ./etc/lighttpd/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
-      - ./client:/var/www/client:cached
-      - ./server:/var/www/server:cached
+      - ./:/var/www:cached
```

Comment:
this is not different to what was there before. I would suggest not to mount the whole thing inside `/var/www` and do something like `./:/` and then symlink `./html` and `./html/api'. That would make it closer to what we have in prod

Reply:
@dayllanmaza What I had before was:
```
- ./:/var/www:cached
```
so I'm reverting it to what it was before.

If we were to do:
`./:/`
then we'd have to create the `var/www/html` directory inside our repo which seems unnecessarily verbose.

We need at least this:
```
- ./html:/var/www/html:cached
- ./server:/var/www/server:cached
```
I don't see what advantage there is to mounting to mounts rather than just mounting the whole thing into `/var/www`

There is only a single mount point on prod witch is `~/` and the webroot is `~/public_html` which we will symlink to `~/tool/html`.

I did just notice that the `require_once` in `index.php` should include `__DIR__` so that it will be `~/tool/server/public/index.php` instead of `~/public_html/../../server/public/index.php` which is obviously not a valid path so I fixed that.

Reply:
It was just a nitpick to be honest. Your point is valid.